### DISCUSSION TITLE
Getting message "Access is Denied" On Superset login page and logout page

### DISCUSSION THREAD

**Discussion by mayanklalwani** [apache/superset#28324] [Q&A / Help]
## Screenshot

![image](https://github.com/apache/superset/assets/79749089/41a216c4-10aa-4d72-8a88-4f0a1fe71233)

## Description

On the login page (also when user logout and gets redirected to login page), the message "Access is Denied: appears. I am using Superset version 3.0.

**Comment by rusackas** [comment]
Moving this to a Q&A discussion, since it's not a bug, but most likely an infra or configuration issue. If you have logs/errors that you can share, that might help. You can also take this to `#debug-together` or `#deploying-superset` on [slack](http://bit.ly/join-superset-slack). Are you sure that user is set up on the staging site the same way it might be set up on your prod site?

**Reply by mayanklalwani** [reply]
yes, the setup is same as that of prod site. The only thing is prod still runs on 2.0.0 and we are trying to upgrade to 3.0.0. We started with upgradation on staging and facing this issue.

**Comment by paulsonkevgit** [comment]
Is there an update on the issue i am also facing the same problem
